src/CitizenProject/Voter/CreateCitizenProjectVoter.php

```diff
@@ -17,30 +17,27 @@ public function __construct(CitizenProjectManager $manager)
         $this->manager = $manager;
     }
 
-    public function vote(TokenInterface $token, $subject, array $attributes)
+    protected function supports($attribute, $subject): bool
     {
-        $adherent = $token->getUser();
-
-        if (null !== $subject || !$adherent instanceof Adherent) {
-            return self::ACCESS_ABSTAIN;
-        }
+        return CitizenProjectPermissions::CREATE === $attribute && null === $subject;
+    }
 
-        if (!in_array(CitizenProjectPermissions::CREATE, $attributes, true)) {
-            return self::ACCESS_ABSTAIN;
-        }
+    protected function voteOnAttribute($attribute, $subject, TokenInterface $token): bool
```

Comment:
the method signature needs to be the same as its parent:
`public function vote(TokenInterface $token, $subject, array $attributes)`